src/main/java/mekanism/common/integration/multipart/MultipartTileNetworkJoiner.java

```diff
@@ -0,0 +1,83 @@
+package mekanism.common.integration.multipart;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+import io.netty.buffer.ByteBuf;
+import mcmultipart.api.container.IMultipartContainer;
+import mcmultipart.api.slot.EnumFaceSlot;
+import mcmultipart.api.slot.IPartSlot;
+import mcmultipart.api.world.IMultipartBlockAccess;
+import mekanism.common.base.ITileNetwork;
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.world.IBlockAccess;
+
+public class MultipartTileNetworkJoiner implements ITileNetwork
```

Comment:
When a capability is requested from a multipart container that request is passed on to the parts it contains. When multiple parts have the same capability a `Joiner` (as defined by MCMP) is required, or else only the first part the container finds is used.

Since it's possible for multiple parts implementing the `Capability<ITileNetwork>` capability to exist in the same container (both a cable and a glow panel) a joiner is necessary to ensure that packets arrive at their appropriate destination.

From `mekanism.common.network.PacketTileEntity.java`: 
```
PacketHandler.handlePacket(new Runnable() {
@Override
public void run()
{
    TileEntity tileEntity = message.coord4D.getTileEntity(player.world);
    
    if(CapabilityUtils.hasCapability(tileEntity, Capabilities.TILE_NETWORK_CAPABILITY, null))
    {
        ITileNetwork network = CapabilityUtils.getCapability(tileEntity, Capabilities.TILE_NETWORK_CAPABILITY, null);
        
        try {
            network.handlePacketData(message.storedBuffer);
        } catch(Exception e) {
            e.printStackTrace();
        }
        
        message.storedBuffer.release();
    }
}
}, player);
```

When the `TileEntity` at the message location is a multipart container with 2 or more parts that have the `ITileNetwork` capability MCMP requests the joiner that was registered for this capability. This creates a new `MultipartTileNetworkJoiner` which builds a map of which side (or center) each part in the container is attached to. It uses this map combined with the ordinal headers added to the possible multipart tile entities to route packets correctly.

I'm just heading out the door right now but I'll add some JavaDoc detailing this when I'm back.